lib/logmsg/timestamp-serialize.c

```diff
@@ -25,15 +25,15 @@
 #include "timestamp-serialize.h"
 
 static gboolean
-_write_log_stamp(SerializeArchive *sa, const LogStamp *stamp)
+_write_log_stamp(SerializeArchive *sa, const UnixTime *stamp)
```

Comment:
I accidentally commented stg into the other PR, commenting here too to have everything in one place.

Actually, I believe LogStamp is a better name than unixtime. In case of unixtime, one would associate the "unix epoch", the number of seconds since 1970 january 01, however now that includes timezone and microseconds information too, which can be confusing.
Syslog-ng also has an unixtime macro, that is expanded into "unix epoch", and not printing the timezone. 

Using LogStamp as name would be more clear in my opinion. It would reduce the size of the diff too.

Related to this, the original logstamp module module was eliminated, and unixtime was put in place. That seems also unnecessary for me.

Reply:
Agreeing to https://github.com/balabit/syslog-ng/pull/2483#issuecomment-462163233, marking resolved.